### PR TITLE
Be able to highlight day

### PR DESCRIPTION
**Pull request by sideroad**
Be able to specify highlight day.
Currently, I specified yellow color as highlighted.
If there is any better color, Plz let me know.

https://github.com/airbnb/react-dates/issues/36

### CODE REVIEW

File: css/variables.scss [hunk 1]
```diff
@@ -10,6 +10,7 @@ $react-dates-color-primary-shade-1: #33dacd !default;
 $react-dates-color-primary-shade-2: #66e2da !default;
 $react-dates-color-primary-shade-3: #80e8e0 !default;
 $react-dates-color-primary-shade-4: #b2f1ec !default;
+$react-dates-color-primary-shade-5: #ffe8bc !default;
```
Comment (majapw) [inline css/variables.scss]: I think since this isn't a shade of `react-dates-color-primary` we should name this `react-dates-color-highlighted`

**Comment by Kriget** [comment]
would really love this feature. could this feature be merged into master in the foreseeable future?

**Comment by majapw** [comment]
As soon as the comments are addressed, I think we can move forward.

**Comment by sideroad** [comment]
Sorry for late response.

Here is a screenshot for it!

**Comment by majapw** [comment]
Awesome! Looks great to me. :D Can you rebase squash the commits down into 1 @sideroad? I'll merge it in after that! 👍

**Comment by majapw** [comment]
Hi @sideroad! Instead of pressing the `Update branch` button in the UI, can you run `git rebase master` and then `git rebase -i` in the terminal to squash the commits down into one commit?

**Pull request merged**